src/client/index.ts

```diff
@@ -203,6 +202,7 @@ class Client extends EventEmitter {
    * @param server - URL of the server to connect to.
    * @param options - Options for client settings.
    */
+  // eslint-disable-next-line max-lines-per-function -- Client needs to space for initializing subscriptions
```

Comment:
don't bother adding these eslint comments - the rule will be edited a bit later and that'll handle this error